server/middleware/getUsername.js

```diff
@@ -0,0 +1,11 @@
+const jwtDecode = require('jwt-decode');
+
+const getUsername = (request, response) => {
+	const accessToken = request.headers.cookie.split('=')[1];
```

Comment:
I guess you wrote this before the magic of cookie-parser

Reply:
Hmm. That should have been changed to cookie-parser. Will go and check.

Reply:
Sorted! :)